docs/main/docs/resources/troubleshooting.md

```diff
@@ -26,36 +25,24 @@
+- **INVALID_ORDER_TYPE:** This error means an unsupported or invalid order type was specified.
+- **INVALID_ASSET:** This error occurs when a specified asset is not supported by the contract.
+- **INVALID_AMOUNT:** This error indicates that the provided amount does not match the expected value for the operation.
+- **AMOUNT_OUT_OF_RANGE:** This error signifies that the provided amount is outside the minimum or maximum allowed range for the asset.
+- **INVALID_COUNT:** This error occurs when an invalid count (i.e zero) is provided for a batch operation.
+- **ESCROW_ORDER_ALREADY_EXISTS:** This error indicates that an escrow order with the given ID already exists.
+- **NOT_CLAIMABLE:** This error signifies that the order is not in a state where it can be claimed.
+- **NOT_REFUNDABLE:** This error means the order is not eligible for a refund at this time.
+- **SETTLE_PAYOUT_WITH_FEES_FAILED:** This error indicates that the process of settling a payout, including deducting fees, failed.
 
 ### remoteOrder.sol
 
-- **RO#0:** Is related to access control and ensuring the integrity of critical contract operations, such as preventing unauthorised access to functions and the assignment of roles to an invalid address.
-- **RO#2:** Ensures that functions are only executed when the contract is active and not in halted state.
-- **RO#3:** This error enforces role-based access control within the contract, preventing unauthorized parties from executing sensitive functions.
-- **RO#4:** Prevents users from attempting to create orders with unsupported assets.
-- **RO#5:** Ensures that `maxReward` is within the accepted range for the specific `rewardAsset`.
-- **RO#7:** Prevents mismatches between the expected reward and the actual funds provided.
-- **RO#10:** Prevents issues where the order payload might not be correctly recorded, which could lead to disputes or untraceable orders.
-- **RO#11:** Ensures that the claim for a refund is only made after the order has timed out.
-- **RO#12:** Ensures that the payment hash matches the expected hash before processing a refund.
-- **RO#13:** Ensures that the payment hash matches the expected hash before processing a payout.
-- **RO#14:** Ensures that only the `attesters` address can settle surplus rewards.
-- **RO#15 in `confirmBatchOrder`:** Ensures that the total amount of native currency expected by the contract matches the amount actually sent with the transaction - `require(totalNativeAmount == msg.value, "RO#15");`
-- **RO#15 in `claimPayoutBatch`:** Enforces checks related to the validity of the payout claim process.
+- **RO#0:** Ensures that a valid address or range of values was provided during initialization.
+- **RO#2:** Ensures that functions are only executed when the contract is active and not in halted state (i.e. amounts outside allowed range, order's timestamp is invalid, order is already confirmet, etc).
```

Comment:
```suggestion
- **RO#2:** Ensures that requested cross-chain route is enabled (both source & destination chains and assets are enabled).
```suggestion
- **RO#2:** Ensures that functions are only executed when the contract is active and not in halted state (i.e. amounts outside allowed range, order's timestamp is invalid, order is already confirmet, etc).
```